nemo/collections/tts/modules/radtts.py

```diff
@@ -581,13 +582,13 @@ def infer(
         self,
         speaker_id,
         text,
-        sigma,
-        sigma_txt=0.8,
-        sigma_f0=0.8,
-        sigma_energy=0.8,
+        sigma=0.0,
```

Comment:
Can we change the sigma value to 0.7?

Reply:
This is botched diff - it is at 0.7

Reply:
Let's make `forward_for_export` and `infer` have the same default values. In case, let's put `sigma` for both at 0.7

Reply:
Done.